tests/components/zwave_js/test_api.py

```diff
@@ -936,12 +937,46 @@ async def test_add_node(
         assert msg["error"]["code"] == "zwave_error"
         assert msg["error"]["message"] == "zwave_error: Z-Wave error 1 - error message"
 
+    # Test inclusion already in progress
+    client.async_send_command.reset_mock()
+    type(client.driver.controller).inclusion_state = PropertyMock(
+        return_value=InclusionState.INCLUDING
+    )
+
+    # Create a node that's not ready
+    node_data = deepcopy(nortek_thermostat.data)  # Copy to allow modification in tests.
+    node_data["ready"] = False
+    node_data["values"] = {}
+    node_data["endpoints"] = {}
+    node = Node(client, node_data)
+    client.driver.controller.nodes[1] = node
```

Comment:
Node ID 1 is the controller node ID. That's reserved for the controller. We should try and keep the tests realistic.